python/xorbits/core/data.py

```diff
@@ -177,17 +175,60 @@ def __setattr__(self, key, value):
         else:
             self.data.__setattr__(key, value)
 
+    def own_data(self):
```

Comment:
IMO this kind of method should not be exposed to public, at least for now, we can use `_own_data` instead.